ql/grammar.go

```diff
@@ -40,6 +40,7 @@ type Query struct {
 	Between      *Between     `@@?`
 	Select       Select       `@@`
 	Where        *Expression  `("where" @@)*`
+	Descending   bool         `@"descending"?`
```

Comment:
@jainilajmera let's change this to "desc". That is consistent with SQL and probably will be more in line with expectations.

Reply:
the string that is, not the field name (which is good).